setup.py

```diff
@@ -114,4 +114,4 @@
             ]
         ],
     ),
-)
+)
```

Comment:
Interesting... if a human comments on a PR review made by the Lintly bot, then fix the issue, it will create this odd detached comment once the linting issue is fixed.

Reply:
It looks like a better way of going about this is to use PR annotations via the [`--use-checks`](https://github.com/grantmcconnaughey/Lintly/issues/36#issuecomment-701149548) option.